breathe/parser/compoundsuper.py

```diff
@@ -798,6 +798,8 @@ def buildChildren(self, child_, nodeName_):
 class incType(GeneratedsSuper):
     subclass = None
     superclass = None
+    # This is an empty list used to bypass render_iterable() in visit_docreftext()
+    para = []  # type: ignore
```

Comment:
I think it is fairly subjective but I would lean towards removing this and adding a check to the `visit_docreftext` function to see if there is a `para` attribute on the node in question.

I'm generally in favour of this kind of approach where you try to minimise if-statements in the code but I don't think that `visit_docreftext` should assume the presence of `para`, I suspect it is a historical assumption that is being proven wrong by this change. Unless for some reason it makes sense to think of incTypes as having paragraphs.

Reply:
Good feedback!

Reply:
> I don't think that visit_docreftext should assume the presence of para, I suspect it is a historical assumption that is being proven wrong by this change.

I'm not sure when in the past `visit_docreftext()` was ever used. I don't see any other direct calls to this function - possibly called via `render(method.get(node.type, visit_unknown())` at some point (presently).

I added a `if hasattr(node, "para")` condition to `visit_docreftext()` and removed my hack from _compoundsuper.py_. It seems to work just as well.